group/urls.py

```diff
@@ -3,5 +3,8 @@
 from . import views
 
 urlpatterns = [
-    path('', views.create_group)
+    path('', views.create_group),
+    path('user/', views.add_users),
+    path('delete/', views.delete_group),
```

Comment:
@APIYOJENNIFER yes, Thatcher is correct. They are actually called path parameters. These are needed for actions that will be performed on a single resource (read group in this case) that already exists on the server. I would advise you read up more about how RESTful APIs work to get a good grasp of that first, otherwise you will end up designing systems the wrong way.